crates/orchestrator/src/tests/database/mod.rs

```diff
@@ -1,7 +1,64 @@
+use crate::config::config;
+use crate::jobs::types::{ExternalId, JobItem, JobStatus, JobType};
+use crate::tests::common::drop_database;
+use crate::tests::config::TestConfigBuilder;
 use rstest::*;
+use uuid::Uuid;
 
 #[rstest]
 #[tokio::test]
-async fn test_database() {
-    // TODO: write test case
+async fn test_database_connection() -> color_eyre::Result<()> {
+    TestConfigBuilder::new().build().await;
+    Ok(())
+}
+
+/// Tests for `create_job` operation in database trait.
+/// Creates 3 jobs and asserts them.
+#[rstest]
+#[tokio::test]
+async fn test_database_create_job() -> color_eyre::Result<()> {
+    TestConfigBuilder::new().build().await;
+
+    drop_database().await.unwrap();
```

Comment:
Valid, moved to `TestConfigBuilder.build()`.



### IMPORTANT 
[This branch is a fork from [increasing coverage](https://github.com/madara-alliance/madara-orchestrator/tree/feat/increasing-test-coverage) ]
[This comment refers to changed already merged to main, they are shown here since #52 is merged and the base of this PR automatically shifted to `main` ]